# MotorCommandMsg.msg

component_lib/GenericComponentMsg component_property

uint8 current_limit
uint8 pwm

# See definitions in component_lib.h
